asapdiscovery-docking/asapdiscovery/docking/tests/test_retro_docking.sh

```diff
@@ -0,0 +1,10 @@
+# I know we'd rather use pytest, but this was much faster to write
+rm -rf ./outputs/retro_docking_test
+run-docking-oe \
+-l './inputs/Mpro_combined_labeled.sdf' \
```

Comment:
This SDF doesn't seem to be vendored along with the inputs  (can it be?) In my case didn't seem to run correctly without it.

Reply:
this is because we haven't resolved this issue yet: #187